components/script/dom/eventdispatcher.rs

```diff
@@ -105,14 +110,17 @@ fn dispatch_to_listeners(event: &Event, target: &EventTarget, chain: &[&EventTar
     assert!(!event.stop_propagation());
     assert!(!event.stop_immediate());
 
-    /* bubbling */
+    // 8. If event's bubbles attribute value is true, run these substeps:
     if !event.bubbles() {
         return;
     }
 
-    event.set_phase(EventPhase::Bubbling);
```

Comment:
I moved this after the next block (see the comment for 8.2) as I felt this reflected the spec